### PR TITLE
Fix/other stacked notebooks

### PR DESCRIPTION
**Pull request by martins0n**
## Before submitting (must do checklist)

- [ ] Did you read the [contribution guide](https://github.com/tinkoff-ai/etna/blob/master/CONTRIBUTING.md)?
- [ ] Did you update the docs? We use Numpy format for all the methods and classes.
- [ ] Did you write any new necessary tests?
- [ ] Did you update the [CHANGELOG](https://github.com/tinkoff-ai/etna/blob/master/CHANGELOG.md)?

### CODE REVIEW

File: scripts/cache_cleaner.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import json
```
Comment (alex-hse-repository) [inline scripts/cache_cleaner.py:1]: Where it is planned to use this script?

**Review by alex-hse-repository — CHANGES_REQUESTED** [review]
Did you ran the tests for notebooks and check that everything is all right now?

**Comment by martins0n** [comment]
> Did you ran the tests for notebooks and check that everything is all right now?

https://github.com/tinkoff-ai/etna/actions/runs/3702532515/jobs/6276086714

File: scripts/cache_cleaner.py (hunk 1 above)
Comment (martins0n) [reply scripts/cache_cleaner.py:1]: In case of corrupted cache as it was today on master branch -- you can delete cache manually via this script

**Review by alex-hse-repository — APPROVED** [review]

**Pull request merged**